src/ai/backend/kernel/__main__.py

```diff
@@ -15,6 +15,17 @@
 from .compat import asyncio_run_forever
 
 
+def setproctitle(title):
+    # setproctitle package doesn't work for unknown reasons
+    # We don't need a portable implementation, so here is a Linux-only version
+    import ctypes
+
+    libc = ctypes.CDLL("libc.so.6")
```

Comment:
How about using [`ctypes.util.find_library("c")`](https://docs.python.org/3/library/ctypes.html#finding-shared-libraries) to avoid hardcoding the libc version?

Reply:
Sure.